src/Steam.package/STZnSSOSeasideOAuth2DemoPanel.class/instance/handleCallback.st

```diff
@@ -1,8 +1,8 @@
 actions
 handleCallback
 	"This method is invoked when the SSO provider redirects back to this application." 
-
+|t|
 	oauthSession handleAuthenticationCallback: self requestContext request fields.
 	userData := oauthSession getUserData.
-	self application application login loginUsername: (userData at: 'given_name') password: 'none'.
-	self session userInfos addAll: userData .
+	self session userInfos: (STOSSUserInfos with: oauthSession infos: userData).t:=self session.1haltOnce.  
```

Comment:
You have a `haltOnce` lost there.
And this line has two statements. Maybe you can format your code?